### ISSUE TITLE
Defining the list of ONET codes of interest

### ISSUE THREAD

**Issue by Nosferican** [uva-bi-sdad/stem_edu#24]
Hi @alyssafowers,

Could you summarize how `data/stem_edu/working/Rothwell_STW_list.csv` was generated? A few sentences on the approach / judgment calls / motivation should suffice.

Thanks.

**Comment by alyssafowers** [comment]
Hi @Nosferican,

I based this on [Rothwell's definition of the skilled technical workforce](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2709141), as laid out in this paper. He defines a skilled technical occupation as one that requires a high level of knowledge in a technical domain and does not require a bachelor's degree for entry. He uses [O*NET data](https://www.onetcenter.org/database.html#individual-files) to determine knowledge levels needed for fields/prevalence of bachelor's degree in different fields. I used Rothwell's cutoffs (knowledge scores of at least 4.5/7 and 50% or more of the workers in that field did not have a bachelor's degree). My results might be somewhat different from his because O*NET survey data is updated over time (I used O*NET version 23.3, while he used version 19).

The knowledge domains that he considered "technical knowledge" included biology, building and construction, chemistry, computers and electronics, design, economics and accounting, engineering and technology, food production, mathematics, mechanical, medicine and dentistry, physics, production and processing, and telecommunications.

The code is in [src/ONET_define/defineCodeRothwell.R](https://github.com/uva-bi-sdad/stem_edu/blob/master/src/ONET_define/defineCodeRothwell.R
). There is some data exploration in there, as well as just creating the list, but it's commented throughout so hopefully it makes sense.
 
I followed Rothwell's definition pretty strictly. The main judgment call was deciding to use Rothwell's definition. I had some doubts about the cutoff points (4.5 doesn't make very much sense when you look at the actual distribution of knowledge scores). However, Rothwell's definition is the standard right now, and in the interest of consistency/moving forward with the project, we went with it.

Let me know if there's anything else I can do to help!
